Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -175,28 +155,41 @@ def bump_formula_pr
       new_url.sub "mirrors.ocf.berkeley.edu/debian", "mirrorservice.org/sites/ftp.debian.org/debian"
     end
     new_mirrors ||= [new_mirror] unless new_mirror.nil?
-    forced_version = args.version
+    new_version = args.version
+    old_url = formula_spec.url
+    old_tag = formula_spec.specs[:tag]
+    old_formula_version = formula_version(formula, requested_spec)
+    old_version = old_formula_version.to_s
+    forced_version = false
     new_url_hash = if new_url && new_hash
       true
     elsif new_tag && new_revision
       false
     elsif !hash_type
-      odie "#{formula}: no --tag=/--revision= arguments specified!"
-    elsif !new_url
-      odie "#{formula}: no --url= argument specified!"
+      odie "#{formula}: no --tag= or --version= argument specified!" if !new_tag && !new_version
+      new_tag ||= old_tag.gsub(old_version, new_version)
+      if new_tag == old_tag
+        odie <<~EOS
+          You probably need to bump this formula manually since the new tag
+          and old tag are both #{new_tag}.
+        EOS
+      end
+      resource_path, forced_version = fetch_resource(formula, new_version, old_url, tag: new_tag)
+      new_revision = Utils.popen_read("git -C \"#{resource_path}\" rev-parse -q --verify HEAD")
+      new_revision = new_revision.strip
+      false
+    elsif !new_url && !new_version
+      odie "#{formula}: no --url= or --version= argument specified!"
     else
-      resource = Resource.new { @url = new_url }
-      resource.download_strategy = DownloadStrategyDetector.detect_from_url(new_url)
-      resource.owner = Resource.new(formula.name)
-      if forced_version
-        if forced_version == resource.version
-          forced_version = nil
-        else
-          resource.version = forced_version
-        end
+      new_url ||= old_url.gsub(old_version, new_version)
+      if new_url == old_url
+        odie <<~EOS
+          You probably need to bump this formula manually since the new URL
```

Comment:
Can we drop the `probably` here? It doesn't make it clear to the contributor what they should do.